index.bs

```diff
@@ -523,6 +523,17 @@ The W3C Team</h3>
 	(generally to other individuals in the Team)
 	for any of their roles described in this document,
 	except <a href="#tag-participation">participation in the TAG</a>.</p>
+	Decisions being described in this document
+	as being <dfn noexport lt="Team Decision | Team's Decision">Team Decisions</dfn> derive from the [=Director=] and [=CEO=]'s authority,
+	even when they are carried out by other members of the [=Team=].
+
+	Note: This does not imply that the CEO or Director are expected to make all decisions personally,
```

Comment:
This note seems redundant with the paragraph just above.

Reply:
Well, it's a note, so it's good that it doesn't contradict the normative text, nor adds requirements that aren't otherwise there. So, it's not entirely wrong to describe it as redundant, but I see it as clarifying intent.

Before there was this note, there were concerns (at least from @swickr), that the text above, without a note somewhere along these lines, could be read as a step away from the current culture, and as attempting to make the team much more top-down than it currently is.

Reply:
How about -

Note: The CEO and Director will take the Team's input into consideration when making decisions. They will delegate responsibilities to Team individuals or groups and define the mechanisms by which this delegation is managed. At all times the CEO and Director have the ultimate authority within the Team.

Reply:
As a practical matter, the CEO rarely makes decisions, and almost always decides via a W3M consensus.  The language of the pull request is a bit strange in that it omits W3M.  One approach is a shorter note which merely says "typically, the CEO and Director involve Team members - specifically W3M - for input and internal Team consensus".

Reply:
W3M has never been defined in the process, and I don't think it should be (in line with the comment below from @wseltzer).

The CEO is responsible for the decision - and I suspect is the one who determines when there is a W3M consensus. In which case it makes much more sense that the process states the CEO is responsible for the decisions.